autogpt_platform/frontend/src/components/integrations/credentials-input.tsx

```diff
@@ -695,6 +796,223 @@ export const UserPasswordCredentialsModal: FC<{
   );
 };
 
+export const HostScopedCredentialsModal: FC<{
```

Comment:
It is a good practice to stick to 1 component per file. Can we move this to a separate file in the same directory? 📁